serving-api/src/main/scala/com/stratio/sparkta/serving/api/actor/StreamingActor.scala

```diff
@@ -50,7 +48,6 @@ class StreamingActor(streamingContextService: StreamingContextService,
   override def receive: PartialFunction[Any, Unit] = {
     case CreateContext(policy) => doCreateContext(policy)
     case GetContextStatus(contextName) => doGetContextStatus(contextName)
```

Comment:
"case DeleteContext(contextName) => doDeleteContext(contextName)"

Please, don't remove this message, is necessary for the statuses of the contexts.

In Cluster mode we add the funtionality in the next iterations.